src/network_analysis.jl

```diff
@@ -645,25 +709,43 @@ end
 
 # Used in the subsequent function.
 function cache_conservationlaw_eqs!(rn::ReactionSystem, N::AbstractMatrix, col_order)
+    # Retrieves nullity (the number of conservation laws). `r` is the rank of the netstoichmat.
     nullity = size(N, 1)
-    r = numspecies(rn) - nullity     # rank of the netstoichmat
-    sts = species(rn)
```

Comment:
`sts` kind of suggests `states`, which is no longer a thing. I would change to `us`, but since these are species specifically, I changed this (here and in two calls over the next few lines) to `sps`.